Documentation/contributor-guide/community-membership.md

```diff
@@ -115,6 +115,14 @@ depend on to make decisions in the best interest of the project in a consistent
 - Subscribed to [etcd-maintainers@googlegroups.com]
 - Elected by supermajority of active maintainers.
 
+### Onboarding process
+
+- Open a PR and add an entry to the [MAINTAINERS] file.
```

Comment:
As maintainer requires a supermajority I would make an exception that this should be done privatly.
I would start from asking two maintainers to become a sponsor and sending an email with candidacy to [etcd-maintainers-private@googlegroups.com] asking for a vote. 

Existing maintainers would then vote privatly (move discussion to bbc) allowing a full transparency. After votes are cast one of maintainers would respond with result to the original email. Either positive or negative with feedback for suggested improvement.

If result is positive, candidate can file a public PR

Reply:
+1 @serathius

Reply:
+1 Basically makes sense to me.  

> If result is positive, candidate can file a public PR

Any of the existing maintainers can also raise a public PR after there is a agreement;

Reply:
> Any of the existing maintainers can also raise a public PR after there is a agreement;

Sure, however I would like to follow K8s and have the process driven by candidate so it's doesn't get stuck and the candidate doesn't need to wait for maintainer creating the PR.

Reply:
I think the steps are good, will describe in the file. But IIUC only maintainers can send emails to [[etcd-maintainers-private@googlegroups.com](mailto:etcd-maintainers-private@googlegroups.com)], so a candidate should use [[etcd-maintainers@googlegroups.com](mailto:etcd-maintainers@googlegroups.com)]?